extensions/wikia/FacebookPreferences/modules/ext.wikia.facebookPreferences.js

```diff
@@ -0,0 +1,129 @@
+/* global require, window */
+require(['jquery', 'mw', 'wikia.loader', 'wikia.nirvana', 'BannerNotification'], function ($, mw, loader, nirvana, BannerNotification) {
+
+	function showConnectSuccess() {
+		new BannerNotification()
+			.setContent(mw.message('fbconnect-preferences-connected').escaped())
+			.setType('confirm')
+			.show();
+	}
+
+	function showDisconnectSuccess() {
+		new BannerNotification()
+			.setContent(mw.message('fbconnect-disconnect-info').escaped())
+			.setType('confirm')
+			.show();
+	}
+
+	function showConnectFailure(res) {
+		if (!res) {
+			return;
```

Comment:
Any reason for that? I think a comment here explaining why we don't display error message here would be nice.
Perhaps it showing a generic error is a better decision here?